src/main/java/com/cactus/systems/UserSystem.java

```diff
@@ -0,0 +1,77 @@
+package com.cactus.systems;
+
+import com.cactus.adapters.AuthAdapter;
+import com.cactus.adapters.Response;
+import com.cactus.adapters.Response.Status;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+/***
+ * Represents the system that controls users
+ */
+public class UserSystem {
+    private AuthAdapter userManager;
+    long currentUserId;
+
+    /***
+     * Create a new GroceryListSystem with user and groceryList managers, and mapping of grocery list name
+     */
+    public UserSystem(AuthAdapter userManager){
+        this.userManager = userManager;
+    }
+
+    /***
+     * Given a name and a password, creates a new user and stores the user as current.
+     * It will return false when .create() returns a Response with Status that is not "OK"
+     * telling us that the name was already taken or password was not of the correct form.
+     *
+     * @param name     a String containing the name of the new user
+     * @param username a String containing the username of the new user
+     * @param password a String containing the password of the new user
+     * @return true if a newUser was created
+     */
+    public Response createUser(String name, String username, String password){
```

Comment:
I think the controller should parse the Response it gets, then return _only_ the relevant information, which in this case would be `true` if a new user was created, as opposed to just returning the Response. I'm not sure if that was your intention, since that's what you javadoc says.

Reply:
That's a better idea. Also I forgot to change the docs to I'll do that too